kin-sdk/kin-sdk-lib/src/main/java/kin/sdk/AccountData.java

```diff
@@ -0,0 +1,106 @@
+package kin.sdk;
+
+import java.util.Arrays;
+import java.util.Map;
+import kin.base.responses.AccountResponse;
+import kin.base.responses.AccountResponse.Flags;
+import kin.base.responses.AccountResponse.Signer;
+
+public class AccountData {
+
+    private String publicAddress;
+    private long sequenceNumber;
+    private String pagingToken;
+    private Integer subentryCount;
+    private AccountResponse.Thresholds thresholds;
+    private AccountResponse.Flags flags;
+    private AccountResponse.Balance[] balances;
+    private AccountResponse.Signer[] signers;
+    private Map<String, String> data;
+
+    public AccountData(String publicAddress, long sequenceNumber, String pagingToken, Integer subentryCount,
+        AccountResponse.Thresholds thresholds, Flags flags, AccountResponse.Balance[] balances,
+        Signer[] signers, Map<String, String> data) {
+        this.publicAddress = publicAddress;
+        this.sequenceNumber = sequenceNumber;
+        this.pagingToken = pagingToken;
+        this.subentryCount = subentryCount;
+        this.thresholds = thresholds;
+        this.flags = flags;
+        this.balances = balances;
+        this.signers = signers;
+        this.data = data;
+    }
+
+    public String getPublicAddress() {
```

Comment:
we have an inconsistency in APIs regarding getters, some are without "get" and some are with, please check what can be done. (I think it was introduced with `Transaction` object)

Reply:
Do you mean in general? because it is a task by its own.
And this specific class is now our code so i thing a getter should be prefixed with 'get', no?

Reply:
remove all get prefix (also in `Transaction` if we'll break the API)